runtime/src/main/cpp/Natives.h

```diff
@@ -137,7 +137,7 @@ KInt Kotlin_String_compareTo(KString thiz, KString other);
 KInt Kotlin_String_compareToIgnoreCase(KString thiz, KConstRef other);
 KChar Kotlin_String_get(KString thiz, KInt index);
 OBJ_GETTER(Kotlin_String_fromUtf8Array, KConstRef array, KInt start, KInt size);
```

Comment:
`fromUtf8Array` seems to be an old api and this line should be removed.

Reply:
What is the purpose of this header declaration? It doesn't seem to have a definition.

Reply:
This file have already been refactored in master, unnecessary declarations removed.